Package.swift

```diff
@@ -1,27 +1,38 @@
-// swift-tools-version:5.2
+// swift-tools-version:5.5
 
 import PackageDescription
 
 let package = Package(
   name: "swift-case-paths",
+  platforms: [
+    .iOS(.v13),
+    .macOS(.v10_15),
+    .tvOS(.v13),
+    .watchOS(.v6),
+  ],
   products: [
     .library(
       name: "CasePaths",
       targets: ["CasePaths"]
     )
   ],
   dependencies: [
-    .package(name: "Benchmark", url: "https://github.com/google/swift-benchmark", from: "0.1.0")
+    .package(name: "Benchmark", url: "https://github.com/google/swift-benchmark", from: "0.1.0"),
+    //.package(url: "https://github.com/pointfreeco/xctest-dynamic-overlay.git", from: "0.5.0"),
+    .package(url: "https://github.com/pointfreeco/xctest-dynamic-overlay.git", branch: "spi-current-test-case"),
```

Comment:
We typically omit `.git` in our other packages (_e.g._ in `swift-composable-architecture`), so I think we should here too to avoid weird Package.resolved churn/conflicts.

```suggestion
    //.package(url: "https://github.com/pointfreeco/xctest-dynamic-overlay", from: "0.5.0"),
    .package(url: "https://github.com/pointfreeco/xctest-dynamic-overlay", branch: "spi-current-test-case"),
```